runtime/gc_glue_java/ContractslotScanner.hpp

```diff
@@ -196,6 +198,61 @@ class MM_ContractSlotScanner : public MM_RootScanner
 		reportScanningEnded(RootScannerEntity_OwnableSynchronizerObjects);
 	}
 
+	virtual void
+	scanContinuationObjects(MM_EnvironmentBase *env)
+	{
+		reportScanningStarted(RootScannerEntity_ContinuationObjects);
+
+		/* Only walk MEMORY_TYPE_NEW regions since MEMORY_TYPE_OLD regions would not contain
+		 * any objects that would move during a nursery contract.
+		 */
+		MM_HeapRegionDescriptorStandard *region = NULL;
+		GC_HeapRegionIteratorStandard regionIterator(env->getExtensions()->heap->getHeapRegionManager());
+		while (NULL != (region = regionIterator.nextRegion())) {
+			if ((MEMORY_TYPE_NEW == (region->getTypeFlags() & MEMORY_TYPE_NEW))) {
+				MM_HeapRegionDescriptorStandardExtension *regionExtension = MM_ConfigurationDelegate::getHeapRegionDescriptorStandardExtension(env, region);
+				for (UDATA i = 0; i < regionExtension->_maxListIndex; i++) {
+					MM_ContinuationObjectList *list = &regionExtension->_continuationObjectLists[i];
+					list->startContinuationProcessing();
```

Comment:
like before, we can do without 'list', and it's not making any easier to read nor it's not used later